pgjdbc/src/main/java/org/postgresql/core/v3/ScramAuthenticator.java

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) 2017, PostgreSQL Global Development Group
```

Comment:
This appears to be a new file. The year should be 2024

Reply:
It is a refactor from `pgjdbc/src/main/java/org/postgresql/jre7/sasl/ScramAuthenticator.java` that was moved here, but sure I can set the year as 2024.